src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java

```diff
@@ -116,4 +118,43 @@ public void testPagination01() throws Exception
       assertEquals(830, total);
    }
 
+   @Test
+   public void testEmp01() throws Exception
+   {
+      Service service = service();
+      Response res = null;
+
+      res = service.get(url("orders?limit=5&emp(shipregion)"));
+      List<ObjectNode> list = res.data().asList();
+      boolean assertion = true;
+      for (ObjectNode result : list)
+      {
+         if (result.get("shipregion") != null && result.get("shipregion") != "")
```

Comment:
result.get("shipregion") != "" is NEVER valid Java.  You have to use .equals().  != checks object equality, not value equality.  

I would write this this way:

for(ObjectNode result : list)
{
  String shipregion = result.getString("shipregion");
  if(!Utils.empty(shipregion))
    fail("shipregion was supposed to be empty but was: '" + shipregion + "'");
}

There is no reason to continue the loop if any one of the values is not empty.